cmd/session-clear.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"encoding/json"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/minio/pkg/probe"
+)
+
+var sessionClear = cli.Command{
+	Name:            "clear",
+	Usage:           "Clear a previously saved session.",
+	Action:          mainSessionClear,
+	Before:          setGlobalsFromContext,
+	Flags:           globalFlags,
+	HideHelpCommand: true,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+  
+USAGE:
+  {{.HelpName}} [SESSION-ID/all]
+
```

Comment:
Also session id is not optional .. so we should avoid [] and to indicate options you can use | instead of /